app/utils/trayWithText.js

```diff
@@ -0,0 +1,245 @@
+const mergeImg = require('merge-img-vwv')
+const pathToImages = 'app/images/app-icons/numbers/generated-numbers/'
+const pathToCircleImages = 'app/images/app-icons/round-clock/'
+const log = require('electron-log')
+const baseImages = 'app/images/app-icons/numbers/'
+
+class TrayWithText {
+  iconWithNumber (
+    imagePath,
+    minutesToLongBreak,
+    darkModeString,
+    monochrome,
+    platform,
+    invertedMonochromeString
+  ) {
+    const minutesOnTray = minutesToLongBreak
```

Comment:
What behaviour we should see when we go over 99? 

Right now I think we have 49 when we have 149 and such. I believe better would be not to show anything in such cases.

Reply:
I guess minutesToLongBreak are always below 99 as I use modulo in returned. I could show 99. Not showing anything is not really what user would expected(or it would require additional description). Anyway I could easly make it 99 for numbers bigger then 99.

Reply:
Well, seems to that showing nothing is "less wrong" then showing wrong number. 193 and 93 is a big difference. But ok, let's keep it...

Reply:
I take that back, I am testing with 105 minutes and it is showing 5. Let's make it at least 99. But if nothing is easier, let's show nothing

Reply:
I prefer 99 coz user who turn this feature see that it works. empty solution would be less intuitive in my opinion. I just pushed version changes that should make it 99.